README.md

```diff
@@ -23,6 +23,10 @@ Curated phyloreferences produced by the [Curation Tool] as Phyloreference eXchan
 
 We are currently working on a complete workflow that would allow us to [merge separate PHYX files into a single Clade Ontology] available as a single OWL file available for individual download.
 
+## Previous iterations of this code
+
+We initially developed the Clade Ontology in Python before replacing it with a Node.js-based library. That earlier iteration has been tagged as [v0.1] in this repository and is additionally [available from Github as a release].
```

Comment:
```suggestion
We initially developed the Clade Ontology in Python before replacing it with a Node.js-based library. That earlier iteration is available as a release tagged [v0.1] in this repository.
```
This could (and given the importance should) be stated more concisely, see suggestion.